casr/src/bin/casr-cluster.rs

```diff
@@ -419,6 +603,27 @@ fn main() -> Result<()> {
                     INPUT_DIR will be added to OUTPUT_DIR.",
                 ),
         )
+        .arg(
+            Arg::new("update")
+                .short('u')
+                .long("update")
+                .action(ArgAction::Set)
+                .num_args(2)
+                .value_parser(clap::value_parser!(PathBuf))
+                .value_names(["NEW_DIR", "OLD_DIR"])
+                .help(
+                    "Update clusters from OLD_DIR using CASR reports from NEW_DIR",
```

Comment:
"Update clusters IN OLD_DIR..." would be more clear to understand  
Same in usage.md